open-api/rest-catalog-open-api.yaml

```diff
@@ -960,6 +960,15 @@ paths:
           schema:
             type: string
             enum: [ all, refs ]
+        - in: query
+          name: referenced-by
+          description:
+            The fully qualified name of the view (i.e namespace name and view name) that referenced the table.
+            
+            nested namespaces are separated by a unit separator (`0x1F`) byte.
```

Comment:
I think we also need to specify how to join the namespace and the table name, right?

Reply:
yes it does :), added it !